WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/adapters/viewholders/ThreatViewHolder.kt

```diff
@@ -18,6 +19,7 @@ class ThreatViewHolder(
             setTextOrHide(threat_header, threatItemState.header)
             setTextOrHide(threat_sub_header, threatItemState.subHeader)
         }
+        threat_sub_header.setTextColor(threat_sub_header.context.getColorFromAttribute(itemUiState.subHeaderColor))
```

Comment:
Question (❓): I would have used `containerView` to get the context. I know it doesn't matter much, maybe more so for consistency purposes if another view would need another context, to always use the container as an entry point to the context. Why did you choose to get the context from the individual view?

PS: I am just curious... 😄

Reply:
Good question, but I don't have a good answer :). I usually use the containerView when I save the context into a field. When I need the context for a single field, I usually request the context from the field. Moreover, the container_view doesn't even have an id in this layout.